docs/03-tutorials/00-eventing/evnt-02-subs-with-multiple-filters.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Create subscription subscribing to multiple event types
+---
+
+The [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource definition (CRD) is used to subscribe to events. In this tutorial, we will show how to subscribe to one or more event types using the Kyma Subscription.
+
+## Prerequisites
```

Comment:
The prerequisites section is identical for all four tutorials. Will you remember updating them all if any prerequisite changes? You might want to extract the prerequisites to maintain them just once and link to that single source of truth. Basically, a three-step instruction "Set up Eventing" or something like that. Or keep them just in the first tutorial and link to that in the others, something like that:

> ## Prerequisites:
> You have performed the prerequisite steps in tutorial [blabla](xyz).